airflow/providers/databricks/provider.yaml

```diff
@@ -61,7 +61,7 @@ versions:
 dependencies:
   - apache-airflow>=2.6.0
   - apache-airflow-providers-common-sql>=1.10.0
-  - requests>=2.27,<3
+  - requests>=2.27.0,<3
```

Comment:
Maybe also add comment about minimum version? Just in case, if some issue with resolution happen in the future and someone will try to downgrade < 2.26 (very small chance that it would happen).


This one from the HTTP provider

```yaml
    # The 2.26.0 release of requests got rid of the chardet LGPL mandatory dependency, allowing us to
    # release it as a requirement for airflow
```

Reply:
We usually do it only for upper binding, generally speaking it's quite fine to get lower binding from ~ 1/2 years ago without much explanation